src/main/java/org/example/springbootintro/dto/book/BookDto.java

```diff
@@ -0,0 +1,17 @@
+package org.example.springbootintro.dto.book;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+public class BookDto {
```

Comment:
I recommend using `@Data` in controller-dto classes, gives you everything you could ever need at once without any downsights